### PR TITLE
Updated post view as I suggested in #184

### PR DESCRIPTION
**Pull request by CTalvio**
#184 

image

### CODE REVIEW

**Comment by hjiangsu** [comment]
I'll target this to the `develop` branch since thats where all the temp work is being done at the moment!

**Comment by CTalvio** [comment]
Still gonna test that I resolved those conflicts right

**Comment by CTalvio** [comment]
Works for me now.

I also flipped some colors around, so that buttons now pulse the color they become, instead of the one they were

**Comment by hjiangsu** [comment]
Thanks! I'll check it out later today and merge it in if it all looks good :D

File: lib/post/widgets/comment_header.dart [hunk 1]
```diff
@@ -44,16 +46,28 @@ class CommentHeader extends StatelessWidget {
                 ),
                 const SizedBox(width: 8.0),
                 Icon(
-                  myVote == VoteType.down ? Icons.south_rounded : Icons.north_rounded,
-                  size: 12.0 * state.contentFontSizeScale.textScaleFactor,
```
Comment (hjiangsu) [inline lib/post/widgets/comment_header.dart:48]: I would add this line back in to allow to text to scale based on the `textScaleFactor` set in the settings!

`size: 12.0 * state.contentFontSizeScale.textScaleFactor,`

File: lib/post/widgets/comment_header.dart [hunk 2]
```diff
@@ -44,16 +46,28 @@ class CommentHeader extends StatelessWidget {
...
-                  color: myVote == VoteType.up ? Colors.orange : (myVote == VoteType.down ? Colors.blue : theme.colorScheme.onBackground),
+                  Icons.north_rounded,
+                  size: 12.0,
+                  color: myVote == VoteType.up ? Colors.orange : theme.colorScheme.onBackground,
                 ),
                 const SizedBox(width: 2.0),
                 Text(
-                  formatNumberToK(score),
-                  textScaleFactor: state.contentFontSizeScale.textScaleFactor,
```
Comment (hjiangsu) [inline lib/post/widgets/comment_header.dart:54]: Same here, I would add this line back in for scaling purposes

File: lib/post/widgets/comment_header.dart [hunk 3]
```diff
@@ -44,16 +46,28 @@ class CommentHeader extends StatelessWidget {
...
+                const SizedBox(width: 12.0),
+                Icon(
+                  Icons.south_rounded,
+                  size: 12.0,
+                  color: downvotes != 0 ? (myVote == VoteType.down ? Colors.blue : theme.colorScheme.onBackground) : Colors.transparent,
+                ),
+                const SizedBox(width: 2.0),
+                Text(
+                  formatNumberToK(downvotes),
                   style: theme.textTheme.bodyMedium?.copyWith(
```
Comment (hjiangsu) [inline lib/post/widgets/comment_header.dart:69]: I would add in the textScaleFactor set in the other parts mentioned above here!

**Review by hjiangsu — COMMENTED** [review]
Just a few minor changes relating to allowing text to scale based on the text size in the settings page!

**Comment by CTalvio** [comment]
Will do, I should be able to get that done before the end of the day, your time.

**Comment by CTalvio** [comment]
Donezo, also added scaling to the author and community text, and the comment count and post time indicator

**Comment by hjiangsu** [comment]
Awesome, I'll merge this in! Thanks!

**Pull request merged**